content/en/docs/setup/additional-setup/cni/index.md

```diff
@@ -136,6 +136,33 @@ The following table shows the required settings for many common Kubernetes envir
 | Red Hat OpenShift 3.10+ | _(none)_ | _(none)_ |
 | Red Hat OpenShift 4.2+ | `--set components.cni.namespace=kube-system --set values.cni.cniBinDir=/var/lib/cni/bin --set values.cni.cniConfDir=/etc/cni/multus/net.d --set values.cni.chained=false --set values.cni.cniConfFileName="istio-cni.conf" --set values.sidecarInjectorWebhook.injectedAnnotations."k8s\.v1\.cni\.cncf\.io/networks"=istio-cni` | _(none)_ |
 
+#### Istioctl support in 1.4.8+ for OpenShift 4.2+
```

Comment:
I'm not good with English, but I would say something like `Instructions for Istio 1.4 and OpenShift`.

Then I'd add a note stating that Docs for 1.4 are outdated.

Reply:
Done.